File: Basics/Conditional Statements/Exercise/05.py
budget_for_film = float(input())
count_actors = int(input())
price_clothes_for_one_actor = float(input())

price_decor = budget_for_film * 0.10
price_clothes = count_actors * price_clothes_for_one_actor

if count_actors > 150:
    price_clothes = price_clothes * 0.9

all_cost = price_decor + price_clothes
money_left = abs(budget_for_film - all_cost)

if budget_for_film >= all_cost:
    print("Action!")
    print(f"Wingard starts filming with {money_left:.2f} leva left.")
else:
    print("Not enough money!")
    print(f"Wingard needs {money_left:.2f} leva more.")

File: JS-Basics/Conditional Statements/Exercise/05.js
function solve(budget,filmExtras,priceOfClothing) {
    budget *= 0.9
    budget -= filmExtras > 150 ? (filmExtras * priceOfClothing) * 0.9 : filmExtras * priceOfClothing

    let result = budget < 0 
    ? `Not enough money!\nWingard needs ${Math.abs(budget).toFixed(2)} leva more.`
    : `Action!\nWingard starts filming with ${budget.toFixed(2)} leva left.`

    console.log(result)
}
